language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -457,10 +457,7 @@ class DottyLanguageServer extends LanguageServer
     implicit val ctx = driver.currentCtx
 
     val uriTrees = driver.openedTrees(uri)
-    val predicate = (tree: NameTree) => {
-      val sym = tree.symbol
-      !sym.isLocal && !sym.isPrimaryConstructor
-    }
+    val predicate = (tree: NameTree) => !tree.symbol.isLocal
```

Comment:
Maybe we could add `local` as a flag in `Include`, then we wouldn't need a custom predicate here (and as a bonus we could make the implementation smarter and not traverse local trees at all when the `local` flag is not set.